lib/index.js

```diff
@@ -9,31 +10,23 @@ const test = require('./test')
 const updateConfig = require('./update-config')
 const detectFreePort = require('./detect-free-port')
 const mergeConfig = require('./merge-config')
-const _ = require('./utils')
 
 module.exports = co.wrap(function * (options) {
-  const defaultConfigPath = Path.resolve(
-    typeof options.config === 'string' ?
-      options.config :
-      'vue.config'
-  )
+  const configFilePath = path.resolve(options.config || 'vue.config')
 
   let fileConfig
   let devConfig
   let prodConfig
   if (options.config !== false) {
-    const configFilePath = options.config || defaultConfigPath
-    try {
-      fileConfig = require(_.cwd(configFilePath))
+    if (pathExists.sync(configFilePath) ||
+        pathExists.sync(configFilePath + '.js') ||
+        pathExists.sync(configFilePath + '.json')) {
```

Comment:
I'm not sure if for just 3 sync calls, this async solution would be faster enough to justify that complexity.